docs/modules/ROOT/pages/user-guide/bootstrap.adoc

```diff
@@ -34,16 +37,68 @@ public class Configurations {
     }
 }
 ----
+<1> Camel uses the uri scheme to look-up component from its registry wich requires to add the `@Named` annotation to the method otherwise the CDI container would create an anonymous bean and Camel would not be able to look it up.  
```

Comment:
```suggestion
<1> Camel uses the component URI scheme to look-up components from its registry, this requires you to add the `@Named` annotation to the method, otherwise the CDI container would create an anonymous bean and Camel would not be able to look it up.  
```